### PR TITLE
Use absolute paths for async-profiler output files

### PR DESCRIPTION
**Pull request by asodja**
When testing Android Studio sync with gradle-profiler I noticed after a long debugging session that profiler does not work if absolute --output-dir is not set. This fixes it so absolute paths are passed in JVM args.

### CODE REVIEW

File: src/main/java/org/gradle/profiler/asyncprofiler/AsyncProfilerOutputType.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public static AsyncProfilerOutputType from(AsyncProfilerConfig config, GradleSce
     abstract File outputFileFor(ScenarioSettings settings);
 
     File individualOutputFileFor(ScenarioSettings settings) {
-        File outputFile = outputFileFor(settings);
+        File outputFile = outputFileFor(settings).getAbsoluteFile();
```
[the remark this answers is not in the document]
Comment (asodja) [reply src/main/java/org/gradle/profiler/asyncprofiler/AsyncProfilerOutputType.java]: Done.

I also tested jfr, YourKit, JProfiler and they don't have this issue.

File: src/main/java/org/gradle/profiler/asyncprofiler/AsyncProfilerOutputType.java [hunk 2]
```diff
@@ -9,13 +9,13 @@
     STACKS("collapsed") {
         @Override
         File outputFileFor(ScenarioSettings settings) {
-            return settings.profilerOutputLocationFor(".stacks.txt");
+            return settings.profilerOutputLocationFor(".stacks.txt").getAbsoluteFile();
```
Comment (wolfs) [inline src/main/java/org/gradle/profiler/asyncprofiler/AsyncProfilerOutputType.java:12]: I wonder if we should go even deeper: It seems like `ScenarioDefinition.getOutputDir()` or `ScenarioDefinition.outputDir` should always be an absolute file. Then the rest would follow automatically I think.

Comment (asodja) [reply src/main/java/org/gradle/profiler/asyncprofiler/AsyncProfilerOutputType.java:12]: We can do that

**Review by wolfs — APPROVED** [review]
LGTM!

**Pull request merged**